src/setupIntegratedPython.ts

```diff
@@ -42,7 +49,7 @@ export const extractPython = async (directory, pythonPath, onProgress) => {
   if (['linux', 'darwin'].includes(os.platform())) {
     try {
       log.info('Granting perms for integrated python...');
-      await fs.chmod(pythonPath, 0o7777, (error) => {
+      fs.chmod(pythonPath, 0o7777, (error) => {
```

Comment:
Did I not import the promise version of this?

Reply:
Nope.

Reply:
Whoops. In that case we should probably switch it for the promise version right?

Reply:
Sure. One sec